internal/security/setup/option/pkiOption_test.go

```diff
@@ -44,21 +44,37 @@ func TestNewPkiInitOption_GenerateOnly(t *testing.T) {
 	assert.Equal(false, generateOff.(*PkiInitOption).GenerateOpt)
 }
 
-func getMockArguments() []interface{} {
-	var ifc []interface{}
-	// use reflection to find out how many bool type of options in PkiInitOption struct
-	// and create a slice of mock argument interface instances
-	elm := reflect.ValueOf(&PkiInitOption{}).Elem()
-	for i := 0; i < elm.NumField(); i++ {
-		field := elm.Field(i)
-		switch field.Kind() {
-		case reflect.Bool:
-			ifc = append(ifc, mock.AnythingOfTypeArgument("func(*option.PkiInitOption) (option.exitCode, error)"))
-		}
+func TestNewPkiInitOption_CacheOnly(t *testing.T) {
```

Comment:
To make sure the executor instance inside the object has the correct setting with PkiOption, ie `CacheOpt` is set to true for the executor. It is meant to catch the programming error if `CacheOpt` is really set instead of `GenerateOpt` for the executor instance.